config/clock.rb

```diff
@@ -3,4 +3,5 @@
 
 module Clockwork
   every(1.day, "DeleteUnconfirmedEntries.perform_later", at: "00:00")
+  every(30.minutes, "DfE::Analytics::EntityTableCheckJob.set(queue: 'analytics').perform_later")
```

Comment:
The data insights team are suggesting once a day. I don't think we need to send it this often.